airflow2/dags/word2vec_train.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python3
+
+# main.py
```

Comment:
We shall have no DAGs in this project. All of them shall ber in the ML-experiments project.